extensions/amp-analytics/0.1/vendors.js

```diff
@@ -107,6 +107,44 @@ export const ANALYTICS_CONFIG = {
       'xhrpost': false,
       'image': true
     }
+  },
+
+  'chartbeat': {
```

Comment:
Please move this above google analytics. @rudygalfi mentioned that having alphabetical ordering would be good. I forgot to mention this to comscore guys so I'll fix their config but at least you can have it in the right place :)

Thanks!